common/test/acceptance/pages/lms/instructor_dashboard.py

```diff
@@ -400,7 +400,7 @@ def _get_messages(self, title_css, details_css, wait_for_messages=False):
         """
         if wait_for_messages:
             EmptyPromise(
-                lambda: self.q(css=self._bounded_selector(title_css)).results != 0,
+                lambda: len(self.q(css=self._bounded_selector(title_css)).results) != 0,
```

Comment:
Yeah, and rather than an EmptyPromise we should do a wait_for here. But, since I almost flagged this as flaky, I'll take this fix...